src/Page.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Buttons from './Buttons';
+import Counter from './Counter';
```

Comment:
```suggestion
import React from 'react';

import Buttons from './Buttons';
import Counter from './Counter';
```

의미상으로 구분되는 경우 빈 줄을 넣어 구분해주면 어떨까요?

Reply:
고견 감사합니다!